app/controllers/mixins/ems_common_angular.rb

```diff
@@ -241,7 +243,10 @@ def set_ems_record_vars(ems, mode = nil)
         ems.host_default_vnc_port_end = params[:host_default_vnc_port_end].blank? ? nil : params[:host_default_vnc_port_end].to_i
       end
 
-      ems.azure_tenant_id = params[:azure_tenant_id] if ems.kind_of?(ManageIQ::Providers::Azure::CloudManager)
+      if ems.kind_of?(ManageIQ::Providers::Azure::CloudManager)
+        ems.azure_tenant_id = params[:azure_tenant_id]
+        ems.subscription    = params[:subscription].blank? ? nil : params[:subscription]
```

Comment:
This could be:

```ruby
ems.subscription    = params[:subscription] unless params[:subscription].blank?
``` 

Ultimately, this `nil` check should be in the Armrest API.  The problem with having it here is that the ManageIQ REST API (and any other client, really) needs to know about this check.

Something to change in the Armrest API for the next version I guess, @djberg96.

Unless there's some distinct meaning behind empty string vs. `nil` for the Armrest API.